jax/interpreters/xla.py

```diff
@@ -62,6 +62,9 @@
 flags.DEFINE_bool('jax_debug_nans',
                   bool_env('JAX_DEBUG_NANS', False),
                   'Add nan checks to every operation.')
+flags.DEFINE_bool('jax_debug_infs',
+                  bool_env('JAX_DEBUG_infs', False),
```

Comment:
I would use all-caps for the name of the environment variable to match the syntax of the others

Reply:
Whoops, yes, just slipped through – fixed.